ui/src/shared/constants/thresholds.ts

```diff
@@ -119,3 +119,13 @@ export const DEFAULT_THRESHOLDS_LIST_COLORS = [
     value: 0,
   },
 ]
+
+export const DEFAULT_THRESHOLDS_TABLE_COLORS = [
```

Comment:
updated the base table color based on discussions with @alexpaxton on what would look best